framework/db/ActiveRecord.php

```diff
@@ -114,7 +114,13 @@ class ActiveRecord extends BaseActiveRecord
      */
     public function loadDefaultValues($skipIfSet = true)
     {
-        foreach ($this->getTableSchema()->columns as $column) {
+        static $columns = null;
+
+        if ($columns === null) {
+            $columns = $this->getTableSchema()->columns;
+        }
```

Comment:
Such thing may be told about any method. We can in similar improvements at many places. But it does not make much sense. Here the performance can be saved for multiple calls of `loadDefaultValues()`, which is actually a rare use case, while calling `loadDefaultValues()` just a single time, which is common for 'CRUD create action' perfomance decreses. Also you reserve an extra memory for the static variable, which can not be released anyhow untill program termination.